### PR TITLE
Updated EDV title and links

### PR DESCRIPTION
**Pull request by paulgrehan**
Updated Spec to reflect EDV Spec (from Confidential Storage) and modified links in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Community Group (CCG) of the W3C.
 
 ![Vendor Interoperability](https://github.com/decentralized-identity/confidential-storage/workflows/Vendor%20Interoperability/badge.svg)
 
-## Latest [Editors Draft](https://identity.foundation/confidential-storage/)
+## Latest [Editors Draft](https://identity.foundation/edv-spec-storage/)
```
Comment (msporny) [inline README.md:8]:
```suggestion
## Latest [Editors Draft](https://identity.foundation/edv-spec/)
```

Fixing to stay in line w/ repo name.

File: index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Confidential Storage 0.1</title>
+    <title>EDV Specification 0.1</title>
```
Comment (msporny) [inline index.html:4]:
```suggestion
    <title>Encrypted Data Vaults v0.1</title>
```

Spell out the name of the specification.

File: index.html [hunk 3]
```diff
@@ -16,10 +16,10 @@
         specStatus: "unofficial",
 
         // the specification's short name, as in http://www.w3.org/TR/short-name/
-        shortName: "confidential-storage",
+        shortName: "edv-spec",
 
         // subtitle for the spec
-        subtitle: "Confidential Storage",
+        subtitle: "EDV Specification Storage",
```
Comment (msporny) [inline index.html:22]:
```suggestion
        subtitle: "A privacy-preserving, confidential storage and sharing mechanism",
```

**Review by msporny — APPROVED** [review]
LGTM as long as suggested changes are applied.

**Comment by paulgrehan** [comment]
Thanks @msporny - have applied hte suggested fixes

**Review by msporny — APPROVED** [review]

**Comment by OR13** [comment]
This web page seems stale: https://identity.foundation/working-groups/secure-data-storage.html

I would expect to see both EDVs and DWNs listed, with specs and pointers to implementations.

**Pull request merged**